include/rgl/api/core.h

```diff
@@ -97,6 +97,18 @@ typedef struct Node *rgl_node_t;
  */
 typedef struct Scene *rgl_scene_t;
 
+/**
+ * Enumerates available extensions in RGL which can be queried using `rgl_get_extension_info`.
+ * For more details, see the chapter on extensions in the README.
+ */
```

Comment:
Please add a comment that the new extensions must be placed at the end of the enumeration